packages/tool-cache/src/tool-cache.ts

```diff
@@ -367,6 +401,7 @@ async function extractZipNix(file: string, dest: string): Promise<void> {
   if (!core.isDebug()) {
     args.unshift('-q')
   }
+  args.unshift('-o') //overwrite with -o, otherwise a prompt is shown which freezes the run
```

Comment:
do we need to worry about gnu unzip versus bsd? not sure whether that is a think w/ unzip or not.

Reply:
not sure whether unzip is in the gnu toolset or just some other tool

Reply:
we looked and this is fine